store/sqlstore/upgrade.go

```diff
@@ -784,3 +786,14 @@ func upgradeDatabaseToVersion522(sqlStore SqlStore) {
 		saveSchemaVersion(sqlStore, VERSION_5_22_0)
 	}
 }
+
+func upgradeDatabaseToVersion523(sqlStore SqlStore) {
+	//if shouldPerformUpgrade(sqlStore, VERSION_5_22_0, VERSION_5_23_0) {
+	if err := sqlStore.Channel().MigrateChannelsToSidebar(); err != nil {
```

Comment:
This can happen in 5.24. Also, this isn't doing the lazy migration as discussed before. Do we think that the impact of this won't be significant?